.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.pythonPath": "venv/bin/python",
+    "python.linting.pycodestyleEnabled": true,
+    "python.linting.enabled": true,
+    "editor.tabSize": 4,
+    "editor.detectIndentation": false
+}
```

Comment:
This file stores visual studio code settings which should be unrelated to the topic classification repo. If so, let's add it to .gitignore

Reply:
You're right in principle, but I don't think there's anything wrong with keeping IDE setting in sync. I tried to keep them minimal (so they are only the vscode setting for making vscode work _with this repo_). There is not stricly a need to include autopep8 or ipython in the requirements.txt, except that they make development easier, so does this.

So I would prefer to keep them (uwazi also has .vscode/settings.json now :) )

Reply:
okay, agreed.